### DISCUSSION TITLE
Kampvuur sessie 20/09

### DISCUSSION THREAD

**Discussion by KaanKalmi** [KaanKalmi/your-tribe-for-life-squad-page#49] [General]
![image](https://github.com/user-attachments/assets/ca27eb54-6868-4e8b-868e-4d339f3af9c2)

### Vragen die je met je team bespreekt:

• Wat willen jullie opleveren?

• Hoe ver zijn jullie?

• Wat moet er nog gedaan worden?

• Hoe gaat de samenwerking?

• Wat gaat goed?

• wat kan beter?

**Comment by KaanKalmi** [comment]
- **Wat willen jullie opleveren?**
   -  _Een volledig werkende squadpage waarin we dus veel animaties hebben zoals een intro animatie, brief verstuur animatie, card scattering animatie etc._  
   
-  **Hoe ver zijn jullie?**
   - _We hadden het project bijna afgerond, we hadden de core functionaliteiten erin al verwerkt we misde alleen dus nog de animaties en styling._
  
- **Wat moet er nog gedaan worden?**
   - _Animaties toeveogen en de styling volledig finishen, samen met de mediaqueries voor mobiel._
  
- **Hoe gaat de samenwerking?**
   -   _Het ging redelijk goed naast de taken verdeling met deadlines, code conventies en het opstellen van de project structuur. We waten namelijk 'slecht' gestart, iedereen had gelijk een branch met zijn of haar component waardoor we snel tegen problemen aankwamen met merges en routes._

- **Wat gaat goed?**
   - _De samenwerking op school was heel goed, alles werd snel en concreet besproken, we gaven ook snel aan hoever we waren met onze taken, waar we tegen aan liepen etc._

- **Wat kan beter?**
   - _Het opstellen van t project structuur,  deadlines bedenken, code conventies bedenken en globale css klaarleggen zodat iedereen gerust te werk kon gaan en niet telkens tegen problemen aankwam._

**Comment by Yu-Jing-van-Gerwen** [comment]
Wat willen jullie opleveren?

**• Hoe ver zijn jullie?**
de squadpage is bijna af alle must have zaten erin

**• Wat moet er nog gedaan worden?**
niks het project is afgrond

**• Hoe ging de samenwerking?**
de samenwerking ging goed we spraken elkaar op schooltijden en boden elkaar hulp als dat nodig is en we hielden elkaar op de hoogte

**• Wat ging goed?**
iedereen voerde zijn taak uit en we hebben geen ruzie gehad

**• wat kan beter?** 
we hadden een deadline moeten hebben als iemand een taak kreeg
ook is een planning handig geweest want de 1ste code en design review  hadden we nogal weinig
het projectboard beter proberen bij te werken waardoor issues op tijd gesloten worden en we een beter overzicht
hebben wat gedaan is en wat niet

**Comment by Lmikkers** [comment]
## Vragen die je met je team bespreekt:
• Wat willen jullie opleveren?
We willen een squadpagina opleveren die werkt, dit houdt in een squad kunnen kiezen en zien en op een persoon te klikken en informatie van de persoon hiervan zien. Alles uit laden van de API


• Hoe ver zijn jullie?
We hebben de core functionaliteit van de squadpage gemaakt. Je kunt alleen geen mail versturen, omdat er geen mail in de API staat, misschien kunnen we dit koppelen aan github?


• Wat moet er nog gedaan worden?
We hebben alle must afgerond. Voor de volgende keer willen we een begin animatie met een busje. Ook wilde we verschillende animaties toe voegen en wat andere styling.

• Hoe gaat de samenwerking?
De samenwerking ging goed, vooral op school. Online bespraken we vaak wel in het kort wie wat deed, maar dit kon wel wat beter, zoals deadlines af spreken. 

• Wat gaat goed?
Ons samenwerking ging voorspoedig, op school bespraken we hoever iedereen was en waar ieder tegen aan liep en of we elkaar konden helpen. Ook werkten we goed met issues en projectboard.

• Wat kan beter?
We waren gelijk begonnen met componenten aanmaken en pages aanmaken, alleen omdat we dit niet op dezelfde manier hadden gedaan, was dit lastig op het laatst toe te voegen. Dit kunnen we misschien eerder bespreken en een project structuur aan te maken.
Voor de volgende keer handig om een code conventie op te stellen, zodat we dezelfde style aanhouden. 
We hadden geen deadlines afgesproken, hierdoor deden we het meeste op het laatst en dit was nog lastig te mergen, dit is handig om de volgende keer te bespreken en in het projectboard te zetten

**Comment by Robin1224** [comment]
- **Wat willen jullie opleveren?**
We hebben als team een werkende squad pagina kunnen opleveren, waarbij we de core van ons design hebben kunnen realiseren.

- **Hoe ver zijn jullie?**
Het project is grotendeels af. De must-haves zitten er in. We hebben niet alles wat in het originele design zat kunnen verwerken in ons eindproduct maar we hebben wel een werkende pagina kunnen leveren waarin de grote lijnen van het design aanwezig zijn.

- **Wat moet er nog gedaan worden?**
Er zijn nog een heleboel could-haves en should-haves waar we als groep niet aan toegekomen zijn, die eigenlijk wel in het originele design zaten.

- **Hoe gaat de samenwerking?**

  - **Wat gaat goed?**
  Over het algemeen verloopt de samenwerking fijn. Iedereen communiceert duidelijk, vooral op school, en heeft zijn taken afgekregen.

  - **Wat kan beter?**
  Ik denk dat er meer gezamenlijk gewerkt kon worden. Uiteindelijk was het vaak zo dat iedereen een taak op zich nam, en deze gewoon thuis ging maken. Ik denk dat er regelmatig tegen dingen aangelopen werden waar we als groep snel uitgekomen zouden kunnen zijn. Ook denk ik dat er bepaalde momenten waren waarbij we misschien langer gezamenlijk hebben kunnen zitten om meer structuur te creëren in het werkproces, en dat bepaalde van deze meetings een beetje gerushed waren. Ten slotte denk ik dat we als team uitgebreider onze issues hebben kunnen documenteren, maar dit is iets waar wij ook vanuit de docenten al feedback op gekregen hebben.
